network/manager.go

```diff
@@ -382,10 +382,10 @@ func (nm *networkManager) CreateEndpoint(cli apipaClient, networkID string, epIn
 	}
 
 	if nw.VlanId != 0 {
-		// the first entry in epInfo is InfraNIC type
-		if epInfo[0].Data[VlanIDKey] == nil {
+		// leverage passing default index, if failed, call function to find default
```

Comment:
where do we call function to find default index here

Reply:
This was a note for me to try a function call if my original design does not work.